query-engine/prisma-models/src/builders/relation_builder.rs

```diff
@@ -19,6 +21,7 @@ impl RelationBuilder {
             manifestation: self.manifestation,
             model_a_name: self.model_a_name,
             model_b_name: self.model_b_name,
+            relation_mode: self.relation_mode,
             model_a: OnceCell::new(),
             model_b: OnceCell::new(),
             field_a: OnceCell::new(),
```

Comment:
alternatively, we could have the resolved referential actions already in the relation as fields (+ maybe getters) instead of methods, but that doesn't make a real difference

Reply:
Right, but I didn't want to introduce too big of a diff in this PR :)